testutils.py

```diff
@@ -51,3 +51,16 @@ def if_rollback():
     if "ROLLBACK TARGET" in out:
         return True
     return False
+
+def get_fedora_release():
+    "Finds the fedora version in str"
+    ver = ""
+    with open("/etc/os-release") as fobj:
+        data = fobj.readlines()
+
+    for line in data:
+        line = line.strip()
+        if line.startswith("VERSION_ID="):
```

Comment:
Newbies also learn from existing code. Take it as a reference and sometimes consider it as a rule of thumb.